orga/app/styles/components/login-or-register.scss

```
@@ -1,10 +1,17 @@
 .login-or-register {
   margin: 20px 0;
 
+  &__language-switcher {
+    @include device-is('mobile') {
+      margin-left: 22px;
+    }
+  }
+
   &__panel {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+    margin-bottom: 16px;
```

Si les valeurs se trouvent dans les design tokens de PixUI, il vaudrait mieux les utiliser.

```suggestion
    margin-bottom: $pix-spacing-s;
```

---

On discutait du css avec @reibecca et @Thomasevano en début d'aprèm et suite au daily de ce matin (sr des assertions de tests), je proposais de faire un ticket de mise à niveau aux standards pour ne pas allonger encore la durée de ce ticket. Qu'en penses tu ?

---

Ok pour moi.